File: Day 18/Aayushi-Mittal.py
# To print upper triangle matrix.

m = int(input())
n = int(input())
mat=[]
for i in range(0, m): 
  l=[]
  for j in range(0, n): 
    if j<n-i:
      print(f"mat[{i}][{j}] : ")
      e = int(input()) 
    else:
      print(f"mat[{i}][{j}] : 0")
      e = 0
    l.append(e)
  mat.append(l)

for i in range(0, m): 
  for j in range(0, n): 
    print(mat[i][j], end = " ") 
  print()

File: Day 19/Aayushi-Mittal.cpp
// To print a the lower triangle matrix.

#include<iostream>
using namespace std;
int main()
{
  int m=10, n=10, mat[m][n];
  cout<<"Enter the number of rows and column: ";
  cin>>m>>n;
  for(int i=0; i<m; i++)
  {
    for(int j=0; j<n; j++)
    {
      if(i>=j)
      {
        cout<<"mat["<<m<<"]["<<n<<"] :"; 
        cin>>mat[i][j];
      }
      else
        mat[i][j]=0;
    }
  }
  cout<<"mat["<<m<<"]["<<n<<"] :"<<endl;
  for(int i=0; i<m; i++)
  {
    for(int j=0; j<n; j++)
    {
      cout<<mat[i][j]<<" ";
    }
    cout<<"\n";
  }
}
